.github/workflows/gen-db.yml

```diff
@@ -34,26 +35,16 @@ jobs:
 
     - name: Generate sqlite (file) database
       run: |
-        python scripts/tutorials/generate_database.py sqlite
+        python simple/utils/generate_database.py
       working-directory: .
 
-    # The postgres database creation can take a while on the hobby-dev tier in Heroku
-    # Disabling until we have a better idea on how to use this
-#    - name: Generate postgres (Heroku) database
-#      env:
-#        SIMPLE_DATABASE_URL: ${{secrets.SIMPLE_DATABASE_URL}}
-#      run: |
-#        pip install psycopg2
-#        python scripts/tutorials/generate_database.py postgres
-#      working-directory: .
-
     - name: Push database file
       uses: dmnemec/copy_file_to_another_repo_action@main
       # Details for this action at https://github.com/marketplace/actions/push-a-file-to-another-repository
       env:
           API_TOKEN_GITHUB: ${{ secrets.SIMPLE_TOKEN }}
       with:
-          source_file: 'SIMPLE.db'
+          source_file: 'SIMPLE.sqlite'
```

Comment:
Once this is merged, we will need to go to the binary repo and delete the old db file to avoid confusion.